libnetwork/iptables/firewalld.go

```diff
@@ -70,6 +71,12 @@ var (
 func FirewalldInit() error {
 	var err error
 
+	// When running with RootlessKit, firewalld is running as the root outside our network namespace
+	// https://github.com/moby/moby/issues/43781
+	if rootless.RunningWithRootlessKit() {
+		return fmt.Errorf("skipping firewalld management for rootless mode")
+	}
```

Comment:
Instead of producing an error here, would it make sense to make calling it conditional? https://github.com/moby/moby/blob/2b70006e3bfa492b8641ff443493983d832955f4/libnetwork/iptables/iptables.go#L135

Alternatively we could log this as an `info` perhaps? (and return `nil` instead of an error)

Reply:
It's actually being logged as debug: https://github.com/moby/moby/blob/2b70006e3bfa492b8641ff443493983d832955f4/libnetwork/iptables/iptables.go#L109

Reply:
Thx! Yes, I saw that part; but also know that _most_ installs wouldn't (and _shouldn't_) have debug-level logs enabled. 

Give that this (IIUC) is basically an "expected" scenario, it felt like this would be something to log as an `Info` (?) log entry. It felt a bit wrong to ignore "any error" returning from the function (perhaps it's ok though), but also to return an error even though we know it's not really an "error".

Reply:
>  log as an Info (?)

Can it be worked in a separate PR?

Reply:
@AkihiroSuda would this work for you? https://github.com/AkihiroSuda/docker/pull/1 (let me know if that LGTY, and of course feel free to squash)

Reply:
Thanks, merged and squashed